Library/OptimizelyFeature.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 
 public enum OptimizelyFeature: String, CaseIterable {
   case commentFlaggingEnabled = "ios_comment_threading_comment_flagging"
+  case consentManagementDialogEnabled = "ios_consent_management_dialog"
```

Comment:
optimizely has this listed as `ios_consent_management`

Reply:
ah good catch. optimizely has been updated